app/src/main/java/org/mozilla/fenix/wallpapers/WallpaperManager.kt

```diff
@@ -239,11 +239,7 @@ class WallpaperManager(
             Wallpaper.Local.Firefox("cerulean", R.drawable.cerulean),
             Wallpaper.Local.Firefox("sunrise", R.drawable.sunrise),
         )
-        private val remoteWallpapers: List<Wallpaper.Remote> = listOf(
-            Wallpaper.Remote.Focus(
-                "focus",
-            ),
-        )
+        private val remoteWallpapers: List<Wallpaper.Remote> = emptyList()
```

Comment:
I thought we would leave this available in only nightly still, so that we could continue to gather usage data?

Reply:
Yes, we will. 
For instance, this PR is only introducing this commit on the branch [mozilla-mobile:releases_v98.0.0](https://github.com/mozilla-mobile/fenix/tree/releases_v98.0.0), that it's what we are using for Beta, and everything that we have in the [main](https://github.com/mozilla-mobile/fenix/tree/main) branch will be what we have on nightly :) 

<img width="992" alt="image" src="https://user-images.githubusercontent.com/773158/153258816-e7825936-68cf-44f1-a600-834ae3022ff4.png">

When we uplift patches to other version we don't have to introduce the functionality first on `main` and later target branch (that will depend case by case, that will be the ideal situation) :)

Reply:
Ah, I hadn't noticed the target, thanks for pointing that out. Do we merge release branches back into main at any point, to make sure the two are synced up?

Reply:
> Ah, I hadn't noticed the target, thanks for pointing that out

No problem :) 

> Do we merge release branches back into main at any point, to make sure the two are synced up?

We don't. As we only uplift only certain bug and functionalities that we know that are safe. When the branch gets stale or a new version arrive ([merge day](https://github.com/mozilla-mobile/fenix/wiki/Creating-a-release-branch)) we just create a new branch that it's based of main.